stubs/hdbcli/hdbcli/dbapi.pyi

```diff
@@ -4,13 +4,13 @@ from collections.abc import Sequence
 from datetime import date, datetime, time
 from types import TracebackType
 from typing import Any, overload
-from typing_extensions import Literal, TypeAlias
+from typing_extensions import Literal, Self, TypeAlias
 
 from .resultrow import ResultRow
 
 apilevel: str
 threadsafety: int
-paramstyle: tuple[str, ...]
+paramstyle: str | tuple[str, ...]  # hdbcli defines it as a tuple which does not follow PEP 249
```

Comment:
I like the comment, but when it's defined as a tuple, why the `str` alternative?

Reply:
Valid point. I thought that there is a use case setting a value, but there is not